README.md

```diff
@@ -12,57 +12,34 @@ to learn how to set up topology using this role.
 ## Table of contents
 
 * [Requirements](#requirements)
-* [Usage example](#usage-example)
```

Comment:
Have you changed the content of some paragraphs or just moved the content into separate files?